toolchain/check/testdata/class/partial.carbon

```diff
@@ -54,7 +62,12 @@ library "[[@TEST_NAME]]";
 
 class C;
 
+// This diagnostic could be more specific - the type might be non-final, but since we only have a declaration, we don't know.
```

Comment:
I think you should make this comment a TODO. The diagnostic is inaccurate here -- the problem is that the class is not defined, not that it is final.